src/lib/Microsoft.Fx.Portability.Reports.DGML/DGMLOutputWriter.cs

```diff
@@ -26,7 +24,7 @@ public class DGMLOutputWriter : IReportWriter
 
         private readonly DGMLManager dgml = new DGMLManager();
 
-        public void WriteStream(Stream stream, AnalyzeResponse response)
+        public Task WriteStreamAsync(Stream stream, AnalyzeResponse response)
```

Comment:
Nice To Have: Useless change to async. This method always completes synchronously.

Reply:
It was for IReportWriter.WriteStream to IReportWriter.WriteStreamAsync